crates/cheatcodes/src/evm.rs

```diff
@@ -62,6 +63,23 @@ impl Cheatcode for loadCall {
     }
 }
 
+impl Cheatcode for loadAllocsCall {
+    fn apply_full<DB: DatabaseExt>(&self, ccx: &mut CheatsCtxt<DB>) -> Result {
+        let Self { pathToAllocsJson } = self;
+
+        // First, load the allocs file from the provided path.
+        let file = File::open(pathToAllocsJson)?;
+        let allocs: HashMap<Address, GenesisAccount> = serde_json::from_reader(file)?;
```

Comment:
this won't work for the genesis file format, which I think we want to support here
this has nested `allocs: HashMap` objects.

so in order to make this cheatcode work for a genesis.json the user would need to extract that first.
instead we should just support both format here, HashMap and `allocs: HashMap`